ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6394,6 +6414,19 @@ TDTM:
   - MONITORING
 
 
+OTKM:
```

Comment:
rename to OTPM (oil temp pressure monitoring)

Reply:
fixed